gandalf.js

```diff
@@ -4,97 +4,104 @@
  * This plugin performs initialization (like calling mermaid.initialize())
```

Comment:
SCRIPT-This function might benefit from using async/await instead of callbacks for better readability.